ktor-swagger/src/test/kotlin/de/nielsfalk/playground/ktor/swagger/SwaggerTest.kt

```diff
@@ -3,79 +3,94 @@
 package de.nielsfalk.playground.ktor.swagger
 
 import com.winterbe.expekt.should
-import org.jetbrains.ktor.application.install
-import org.jetbrains.ktor.locations.Locations
-import org.jetbrains.ktor.locations.location
-import org.jetbrains.ktor.routing.routing
-import org.jetbrains.ktor.testing.withTestApplication
+import io.ktor.application.install
+import io.ktor.locations.Location
+import io.ktor.locations.Locations
+import io.ktor.routing.routing
+import io.ktor.server.testing.withTestApplication
 import org.junit.Before
 import org.junit.Test
 import java.time.Instant
 import java.time.LocalDate
+import kotlin.reflect.KClass
 import kotlin.reflect.full.memberProperties
 
 data class ToyModel(val id: Int?, val name: String)
 data class ToysModel(val toys: MutableList<ToyModel>)
 
-/**
- * @author Niels Falk
- */
-
-@group("toy")
-@location("/toys/{id}")
+const val toysLocation = "/toys/{id}"
+@Group("toy")
+@Location(toysLocation)
 class toy(val id: Int)
 
-@location("/toys")
+const val toyLocation = "/toys"
+@Location(toyLocation)
 class toys
 
-@location("/withParameter")
+@Location("/withParameter")
 class withParameter
 
 class Header(val optionalHeader: String?, val mandatoryHeader: Int)
 class QueryParameter(val optionalParameter: String?, val mandatoryParameter: Int)
 
 class SwaggerTest {
+    private lateinit var swagger: Swagger
+
     @Before
-    fun setUp(): Unit = withTestApplication {
-        //when:
-        application.install(Locations)
-        application.routing {
-            put<toy, ToyModel>("update".responds(ok<ToyModel>(), notFound())) { _, _ -> }
-            get<toys>("all".responds(ok<ToysModel>(), notFound())) { }
-            get<withParameter>("with parameter".responds(ok<Unit>()).parameter<QueryParameter>().header<Header>()) {}
+    fun setUp() {
+        withTestApplication({
+            install(Locations)
+            install(SwaggerSupport)
+        }) {
+            // when:
+            application.routing {
+                put<toy, ToyModel>("update".responds(ok<ToyModel>(), notFound())) { _, _ -> }
+                post<toys, ToyModel>("create".responds(created<ToyModel>())) { _, _ -> }
+                get<toys>("all".responds(ok<ToysModel>(), notFound())) { }
+                get<withParameter>("with parameter".responds(ok<Unit>()).parameter<QueryParameter>().header<Header>()) {}
+            }
+
+            this@SwaggerTest.swagger = application.swagger.swagger
```

Comment:
Not really a  fan of having `application.swagger.swagger` as a part of the API.
`application.swagger` is `SwaggerSupport`
`application.swagger.swagger` is `Swagger`.

Maybe it would be better to change the name of the feature to `SwaggerUi`?
Then we could have it be `application.swaggerUi.swagger`.

Thoughts?